content/getting-started/white-label-apps.md

```diff
@@ -0,0 +1,96 @@
+
+## How to get started with Codemagic when white labeling ~10 different versions
+
+We would recommend getting familiar with setting up a single version of your app to begin with. Set up your workflow using a codemagic.yaml configuration file and consult the documentation to understand core concepts, such as using environment variables, build triggers, script steps, code signing and publishing. Once you have successfully built and published a version of your app to the stores, continue by adding additional versions to your configuration file. For more information about getting started with the codemagic.yaml please refer to the documentation [here](../yaml/yaml-getting-started/).
+
+## How to set up Codemagic with ~10 different versions
+
+If you have around 10 different versions of your application that you are trying to set up CI/CD for, consider separating each version into a unique workflow. Separating the versions into different workflows allows using different build schemas, code signing settings and publishing destinations. Builds can be triggered on events such as new commits to your source code repository, pull requests or adding a specific tag. It is also possible to trigger workflows based on changes in certain files. See [here](https://docs.codemagic.io/yaml/yaml-conditional-run/) for further details.
+
+For example, for each version of your app, you could create a workflow for it in the codemagic.yaml configuration file.
+
+```yaml
+workflows:
+  version-one:
+    name: Version one
+    instance_type: mac_mini
+    environment:
+      groups:
+        ...
+      vars:
+        ...
+    scripts:
+      ...
+    artifacts:
+      ...
+    publishing:
+      ...
+  version-two:
+    name: Version two
+    instance_type: mac_mini
+    environment:
+      groups:
+        ...
+      vars:
+        ...
+    scripts:
+      ...
+    artifacts:
+      ...
+    publishing:
+      ...
+```
+
+## How to get started with Codemagic when white labeling tens or hundreds of different versions
+
+You should try any scripts on your local machine to test things like authenticating with your own CMS system, or running scripts that change application assets, such as icons or images. You can even use Codemagic's open-source [CLI tools](https://github.com/codemagic-ci-cd/cli-tools) to test features such as code signing and publishing. Once you are confident that your scripts work as expected, you can then set up your workflow on Codemagic.
+## How to set up Codemagic with tens or hundreds of different versions
+
+Users can start new builds using [Codemagic REST API](../rest-api/codemagic-rest-api.md). All data is sent and received as JSON. The REST API supports an optional parameter for environment variables. 
```

Comment:
I'd add an intro sentence here, sth like: "If you are building many different versions, then instead of creating a long codemagic.yaml file with tens of workflows, you can have a single workflow and use the Codemagic API to pass environment variables to change build parameters."